### PR TITLE
docs(date picker): audit fixes

### PR DESCRIPTION
**Pull request by aagonzales**
Closes https://github.com/carbon-design-system/carbon-website/issues/2005

### CODE REVIEW

**Review by janchild — COMMENTED** [review]
@aagonzales this looks great! It might be good to add captions to the first two imaes on the Style tab, just for consistency.

**Review by joshblack — APPROVED** [review]

**Review by laurenmrice — CHANGES_REQUESTED** [review]
These images on the usage tab also need to use text-05 for placeholder text. (might be a problem with your build not updating? but just check them.)

- [ ] Anatomy: time picker
time-picker-anatomy

- [ ] States: validation 
date-picker-validation

- [ ] States: states image (validation input)
Screen Shot 2021-01-13 at 3 49 10 PM

- [ ] Variants: date range picker anatomy
Screen Shot 2021-01-13 at 3 49 35 PM


- [ ] Calendar behaviors: Min max dates
Screen Shot 2021-01-13 at 3 49 44 PM


--------

- [ ] The helper text in the first input should be below the input.
Screen Shot 2021-01-13 at 3 47 53 PM

**Comment by aagonzales** [comment]
Ok fixed! I just forgot to update those on the usage page.

File: src/pages/components/date-picker/style.mdx [hunk 1]
```diff
@@ -67,14 +77,17 @@ while invalid.
...
-![Example of a date picker](images/date-picker-style-1.png)
+![Date picker calendar examples](images/date-picker-style-1.png)
 
-</div>
+</Column>
+</Row>
 
 <Caption>
-  Date picker example using both $field-02 (top) and $field-01 (bottom)
+  Examplea of a single date calendar picker (left) and a date range calendar
```
Comment (janchild) [inline src/pages/components/date-picker/style.mdx:89]:
```suggestion
  Example of a single date calendar picker (left) and a date range calendar
```

File: src/pages/components/date-picker/style.mdx [hunk 2]
```diff
@@ -41,6 +41,16 @@ input color is `$field-02` and is used on `$ui-01` page backgrounds.
...
 
+<Row>
+<Column colLg={8}>
+
+![Date picker input states](images/date-picker-style-6.png)
+
+</Column>
+</Row>
+
+<Caption>Example of date picker input states.</Caption>
```
Comment (janchild) [inline src/pages/components/date-picker/style.mdx:52]:
```suggestion
<Caption>Examples of date picker input states</Caption>
```

**Review by janchild — APPROVED** [review]
Thanks, @aagonzales!

**Review by laurenmrice — APPROVED** [review]
🙌🏻 🎉

**Pull request merged**